### PR TITLE
Create zh-trans:docs/user-journeys/users/cluster-operator/foundational.md

### PR DESCRIPTION
**Pull request by zhangx501**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by zhangx501** [comment]
/assign @dchen1107
Please give this PR approval.

File: content/zh/docs/user-journeys/users/cluster-operator/foundational.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+* [Annotations](/docs/concepts/overview/working-with-objects/annotations/)
+* [ConfigMaps](/docs/tasks/configure-pod-container/configure-pod-configmap/)
+* [Secrets](/docs/concepts/configuration/secret/)
+-->
+* [Pod Overview](/docs/concepts/workloads/pods/pod-overview/)
+  * [Pods](/docs/concepts/workloads/pods/pod/)
+  * [ReplicaSets](/docs/concepts/workloads/controllers/replicaset/)
+  * [Deployments](/docs/concepts/workloads/controllers/deployment/)
+  * [垃圾收集](/docs/concepts/workloads/controllers/garbage-collection/)
+  * [容器图片](/docs/concepts/containers/images/)
```
Comment (markthink) [inline content/zh/docs/user-journeys/users/cluster-operator/foundational.md]: Container Images => 容器镜像

File: content/zh/docs/user-journeys/users/cluster-operator/foundational.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  * [ReplicaSets](/docs/concepts/workloads/controllers/replicaset/)
+  * [Deployments](/docs/concepts/workloads/controllers/deployment/)
+  * [Garbage Collection](/docs/concepts/workloads/controllers/garbage-collection/)
+  * [Container Images](/docs/concepts/containers/images/)
+  * [Container Environment Variables](/docs/concepts/containers/container-environment-variables/)
+* [Labels and Selectors](/docs/concepts/overview/working-with-objects/labels/)
+* [Namespaces](/docs/concepts/overview/working-with-objects/namespaces/)
+  * [Namespaces Walkthrough](/docs/tasks/administer-cluster/namespaces-walkthrough/)
+* [Services](/docs/concepts/services-networking/service/)
+* [Annotations](/docs/concepts/overview/working-with-objects/annotations/)
```
Comment (markthink) [inline content/zh/docs/user-journeys/users/cluster-operator/foundational.md]: Annotations => 注解

Comment (zhangx501) [reply content/zh/docs/user-journeys/users/cluster-operator/foundational.md]: @markthink  
I have changed the zh translation to "注解" in line 123. The word "annotations" in line 108 is the original text.

**Pull request merged**